config/standalone.example.toml

```diff
@@ -177,3 +177,15 @@ parallel_scan_channel_size = 32
 # otlp_endpoint = "localhost:4317"
 # The percentage of tracing will be sampled and exported. Valid range `[0, 1]`, 1 means all traces are sampled, 0 means all traces are not sampled, the default value is 1. ratio > 1 are treated as 1. Fractions < 0 are treated as 0
 # tracing_sample_ratio = 1.0
+
+# standalone/frontend/datanode/metasrv export the metrics generated by itself
+# send metrics to Prometheus remote-write compatible receiver (e.g. `greptimedb`)
+# [remote_write]
```

Comment:
I prefer another name, `remote_write` is confusing and lacks context. How about `system_metric` or `self_monitoring`? cc @killme2008

Reply:
maybe system_metric is more suitable